distributed-process-tests/src/Control/Distributed/Process/Tests/CH.hs

```diff
@@ -1369,6 +1372,116 @@ testExitRemote TestTransport{..} = do
   takeMVar supervisedDone
   takeMVar supervisorDone
 
+testRegistryMonitoring :: TestTransport -> Assertion
+testRegistryMonitoring TestTransport{..} = do
+  localNode <- newLocalNode testTransport initRemoteTable
+  remoteNode <- newLocalNode testTransport initRemoteTable
+  return ()
+
+  -- Local process. Test if local process will be removed from
+  -- registry when it dies.
+  box <- newEmptyMVar
+  runProcess localNode $ do
+    pid <- spawnLocal $ do
```

Comment:
[nitpick] redundant `do`

Reply:
Okay I'll lint over the changes. Although the linter is generally unhappy about *lots* of things, so maybe just this function and then a separate cosmetic PR...